test/test_helper.rb

```diff
@@ -3,6 +3,8 @@
   add_filter "/factories/"
 end
 
+Dir[Rails.root.join("test/support/**/*.rb")].each { |f| require f }
+
```

Comment:
It requires all files in my test support folder. I abstracted a log_in method in the support method so that we can simply call log_in_user(user) in the test and it logs you in.